apps/sms/views/conversation/style/conversation.css

```diff
@@ -157,7 +157,3 @@
 #messages-header-text .ellipsis-dir-fix {
   width: auto;
 }
-
-.conversation-view .panel {
```

Comment:
can you remove the class "conversation-view" in views/conversation/index.html then ?